pages/common/kops.md

```diff
@@ -0,0 +1,24 @@
+# kops
+
+> A tool that create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters from the command line.
+> More informations: <https://github.com/kubernetes/kops/>.
+
+- Create a cluster from the configuration specification:
+
+`kops create cluster -f {{my-cluster.yaml}}`
+
+- Create secret from secret spec file:
```

Comment:
```suggestion
- Create a secret from secret spec file:
```

Reply:
I will change this example with the better one!